test/ptc/e2e/system_test.clj

```diff
@@ -15,10 +15,10 @@
 (defn ^:private timeout
   "Timeout FUNCTION after MINUTES."
   ([] ::timed-out)
-  ([seconds function]
+  ([minutes function]
    (let [cancel (timeout)
          ff     (future (function))
-         result (deref ff (* 60 1000 seconds) cancel)]
+         result (deref ff (* 60 1000 minutes) cancel)]
```

Comment:
Woah!  Agree with you that [there has to be a better way](https://broadinstitute.atlassian.net/browse/GH-1545).

Reply:
That's just a naming fix, but I did have to extend one timeout.

Reply:
Ahh okay, so it was always minutes and previously named wrong.